### DISCUSSION TITLE
👋 I would like to Filter on either or [May 15]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5702] [Discord Threads]
> 🙋‍♂️ **phrawzty**

**Comment by o1lab** [comment]
> 🙋‍♂️ **phrawzty**  
#### For example, this table has a Date and Formula columns in it (among other things), but the Filter dropdown doesn't list them. <:sads:766424663180967986>    
####    
#### Any ideas?  
![image0](https://cdn.discordapp.com/attachments/1096468225333674004/1096468856228282540/image.png)  
![image1](https://cdn.discordapp.com/attachments/1096468225333674004/1096468856488345610/image.png)  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Strange. I tried it on PG- was able to see formula & date filters properly.  
![image0](https://cdn.discordapp.com/attachments/1096468225333674004/1096650645261520896/Screenshot_2023-04-15_at_9.47.52_AM.png)  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Do help us if you see any logs?  
#  
> 🙋‍♂️ **wkw**  
#### <@514053134176354319> try scrolling down the list  
#  
> 🙋‍♂️ **phrawzty**  
#### <@495466997304262658> oh my god I'm an idiot. The lack of a scroll bar in the interface made it look like it wasn't a scrollable entity.   
####    
#### Thanks for the (embarrassing) help 😊  
#  
> 🙋‍♂️ **wkw**  
#### we will add the scrollbar  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1096468225333674004)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
